NDB.Covid19/NDB.Covid19/Models/SelfDiagnosisSubmissionDTO.cs

```diff
@@ -29,6 +31,7 @@ public SelfDiagnosisSubmissionDTO()
             Platform = ServiceLocator.Current.GetInstance<IDeviceInfo>().Platform.ToString();
             Regions = Conf.SUPPORTED_REGIONS.ToList();
             IsSharingAllowed = LocalPreferencesHelper.AreCountryConsentsGiven;
```

Comment:
Did you check a scenario that if the person has at some point in the past reported infected with lookup in MSIS and decided to share the keys with EU, then this will be already set to true, so if the same user reports infected with self-test, the users' keys will be marked for sharing with EU?